SPECS/grub2/grub2.spec

```diff
@@ -46,6 +46,8 @@ Patch0156:      0156-efilinux-Fix-integer-overflows-in-grub_cmd_initrd.patch
 Patch0157:      0157-linuxefi-fail-kernel-validation-without-shim-protoco.patch
 # Fix to prevent user from overwriting signed grub binary using grub2-install
 Patch0166:      0166-grub-install-disable-support-for-EFI-platforms.patch
+# CVE-2021-3981
+Patch0167:      0167-restore-umask-for-grub-config.patch 
 # Add nopatches for tooling
 Patch1000:      CVE-2021-3418.nopatch
 Patch1001:      CVE-2020-14372.nopatch
```

Comment:
While you're here- can we remove the `.nopatch` files from this spec, to be consistent with how we handle all new `.nopatch` files?

Reply:
What's our policy of handling .nopatch files? I'm stilling seeing a number of nopatch files existing in other specs?

Reply:
We don't need to mention them in the spec file anymore. Our tooling will pick up on the fact that they're in the same folder as the spec. It's a relatively new thing, I think, so there are still remnants of nopatch files in specs. I won't block on this, it's just a good bit of cleanup to do.

Recent example of a PR where nopatches were added but not added to the spec: #3399

Reply:
I see. Will remove them